sockeye/encoder.py

```diff
@@ -1170,4 +1214,4 @@ def get_encoded_seq_len(self, seq_len: int) -> int:
         return int(ceil(seq_len / self.pool_stride))
 
 
-EncoderConfig = Union[RecurrentEncoderConfig, transformer.TransformerConfig, ConvolutionalEncoderConfig]
+EncoderConfig = Union[RecurrentEncoderConfig, transformer.TransformerConfig, ConvolutionalEncoderConfig, 'ImageLoadedCnnEncoderConfig']  # type: ignore
```

Comment:
It doesn't. But I cannot use the class `ImageLoadedCnnEncoderConfig` instead of the string, because it will be imported in `get_encoder`. This was a trick to avoid circular imports.
Any idea on how to fix this?

Reply:
Ok I solved.
```
ImageEncoderConfig = None

def get_encoder(config: 'EncoderConfig', prefix: str = '') -> 'Encoder':
    ...
    ImageEncoderConfig = ImageLoadedCnnEncoderConfig
    ...

EncoderConfig = Union[RecurrentEncoderConfig, transformer.TransformerConfig, ConvolutionalEncoderConfig] 
if ImageEncoderConfig is not None:
    EncoderConfig = Union[EncoderConfig, ImageEncoderConfig]
```